src/index.js

```diff
@@ -9,5 +9,58 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+import Clue from '../src/Clue'
+import '../src/Game'
+import '../src/Player'
+import '../src/Round'
+// import { promises } from 'dns';
 
-console.log('This is the JavaScript entry file - your code begins here.');
+let clue;
+let clueCategories = [];
+let clueInfo = [];
+
+
+
+function categoryFetch() {
+ return fetch('https://fe-apps.herokuapp.com/api/v1/gametime/1903/jeopardy/data')
+  .then(data => data.json())
+  .then(data => data.data.categories)
```

Comment:
You should technically be able to put your logic from this `.then()` into the next one (ie just call `Object.keys(data.data.categories)`)